integration_tests/suite/base/test_users.py

```diff
@@ -2417,3 +2417,59 @@ def test_update_incall_new_incall():
         ),
     )
     url.delete(recursive=True)
+
+
+@fixtures.device()
+@fixtures.device()
+def test_post_lines_same_unallocated_device_no_error(device, device2):
```

Comment:
Why testing device here? The use case is not "on the same device" but "on the same extension"

Reply:
function renamed